lib/kdi/kdi_helpers.rb

```diff
@@ -154,8 +154,8 @@ def create_kdi_asset_finding(asset_hash, finding_hash, match_key = nil)
         a = find_or_create_kdi_asset(asset_hash, match_key)
 
         # Default values & type conversions... just make it work
-        finding_hash["triage_state"] = "new" unless finding_hash["triage_state"]
-        finding_hash["last_seen_at"] = Time.now.utc.strftime("%Y-%m-%d") unless finding_hash["last_seen_at"]
+        finding_hash["triage_state"] ||= "new"
+        finding_hash["last_seen_at"] ||= Time.now.utc.strftime("%Y-%m-%d")
```

Comment:
This is the first time I have seen this syntax. I did not know you could do that.